### PR TITLE
Update maintainer

### PR DESCRIPTION
**Pull request by ijnek**
Maintenance handover (#111)

Should be backported to foxy-devel and humble, so I receive buildfarm failure emails.

### CODE REVIEW

**Comment by ijnek** [comment]
PR failing due to #113

File: image_transport_plugins/package.xml [hunk 1]
```diff
@@ -12,12 +12,14 @@
...
-  <author>Patrick Mihelich</author>
-  <author>Julius Kammerl</author>
-  <maintainer email="michael@openrobotics.org">Michael Carroll</maintainer>
+  <maintainer email="kenjibrameld@gmail.com">Kenji Brameld</maintainer>
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/image_transport_plugins</url>
+  <author>Patrick Mihelich</author>
+  <author>Julius Kammerl</author>
+  <author email="dgossow@willowgarage.com">David Gossow</author>
```
Comment (mjcarroll) [inline image_transport_plugins/package.xml]: I think at this point, it's safe to drop the `@willowgarage` emails.

```suggestion
  <author>David Gossow</author>
```

File: theora_image_transport/package.xml [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
     transparently sending an image stream encoded with the Theora codec.
   </description>
-  <maintainer email="jkammerl@willowgarage.com">Julius Kammerl</maintainer>
+  <maintainer email="kenjibrameld@gmail.com">Kenji Brameld</maintainer>
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/image_transport_plugins</url>
   <author>Patrick Mihelich</author>
   <author>Ethan Dreyfuss</author>
+  <author email="jkammerl@willowgarage.com">Julius Kammerl</author>
```
Comment (mjcarroll) [inline theora_image_transport/package.xml]:
```suggestion
  <author>Julius Kammerl</author>
```

**Comment by ijnek** [comment]
Thanks @mjcarroll , all addressed.

**Review by mjcarroll — APPROVED** [review]

**Comment by ijnek** [comment]
Merging, as CI failure is unrelated.

**Comment by ijnek** [comment]
@Mergifyio backport humble foxy-devel

**Pull request merged**